### PR TITLE
Prep 0.7.0

### PR DESCRIPTION
**Pull request by RondeauG**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] (If applicable) Documentation has been added / updated (for bug fixes / features).
- [ ] (If applicable) Tests have been added.
- [ ] This PR does not seem to break the templates.
- [ ] HISTORY.rst has been updated (with summary of main changes).
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.

### What kind of change does this PR introduce?

* ...

### CODE REVIEW

**Comment by RondeauG** [comment]
Do we need anything else before we're good to go ?

File: HISTORY.rst
Comment (Zeitsperre) [inline HISTORY.rst:1]: On line 28, there's a space missing (note the green):
``` restructuredText
* Add more comments in the template. (:pull:`233`,:pull:`235`, :issue:`232`).
```

**Review by Zeitsperre — APPROVED** [review]

**Comment by Zeitsperre** [comment]
Nope, I think we're in the clear!

**Review by aulemahal — APPROVED** [review]
Rien à dire de mon bord non plus!

**Pull request merged**